packages/textlint-config/readme.md

```diff
@@ -0,0 +1,31 @@
+# @strv/textlint-config
+
+> STRV's config for [textlint][textlint-home]
+
+## STRV Config
+
+.textlintrc or .textlintrc.json
+
+```json
+{
+  "rules": {
+    "@strv/textlint-rule-preset-strv": true
```

Comment:
Does this have to end with an `-strv` suffix in the package name...? It's already clearly STRV-made based on the npm scope. I would prefer to drop it if possible. 🙏

Reply:
Personally, I don't care. I was just following the convention which says:

> ### Package Naming Conventions
> textlint rule package naming should have textlint-rule-preset- prefix.
> textlint-rule-preset-<name>
> @scope/textlint-rule-preset-<name>

If we want to do it differently it's fine by me 🤷‍♂

Reply:
I read their docs and this seems to have implications - let's leave the package's name `@strv/textlint-rule-preset-strv`. Please change the folder names and readme title to match this name. 🙏